lib/renter.js

```diff
@@ -650,6 +661,8 @@ Renter.prototype._publishContract = function(contacts, contract, callback) {
 
         this._logger.warn('Error publishing contract to node: %j, reason: %s',
                           contact, err.message);
+
+        seenInLastDay(contactInst);
```

Comment:
This won't be saved/persisted. This is modifying the instance of `Contact` from storj-lib, and not the database model.

Reply:
The instance that needs to be modified is the mongoose model that's return from a database query, which can be done here: https://github.com/nat-ventura/complex/blob/8834c59111fb1f40b40f5384d4cf363ce56e7f93/lib/renter.js#L585-L606